docs/projects/tcet-linux/installation/installation-in-drive.md

```diff
@@ -0,0 +1,55 @@
+---
+title: In Drive Istallation Steps
+id: installation-in-drive
+description: Installation Steps of Tcet Linux using a Bootable Drive
+sidebar_label: In-Drive Installation Steps
+keywords: [Installation Steps, Installation, Steps, Guide, linux guide ,  bootable drive]
+---
+TCET Linux Installation `element` is a step-by-step guide created by us that will walk you through the process of installing [**TCET Linux**](https://github.com/tcet-opensource/tcet-linux/releases/download/v0.5-beta/checksum)  using a bootable drive on your system.This `Section` in the [**TCET Linux Website**](https://linux.tcetmumbai.in/) helps the user to easily install `TCET Linux` on their Device.
+
+### Step 1. Download the latest version of TCET LINUX
+
+Download the latest version from our [**official website**](https://www.tcetlinux.com/downloads) .
+
+<!-- ![TCET LINUX](/docs/projects/tcet-linux/assets/tldown.png) -->
+<!-- Above image routing is incorrect, refer the route below and update all image routes -->
+![TCET LINUX](../assets/tldown.png)
+### Step 2. Prepare the Bootable USB Stick
+
+- You need to create a bootable USB stick from which you will install your new operating system. You can use various tools for this purpose such as Rufus.
+
+<!-- ![TCET LINUX](/docs/projects/tcet-linux/assets/rufus.png) -->
```

Comment:
Broken route has been commented. Apply the fix from line 16 and remove the commented code.